projects/plugins/crm/src/automation/class-automation-engine.php

```diff
@@ -203,19 +211,19 @@ public function get_registered_step( $step_name, array $step_data = array() ): S
 	}
 
 	/**
-	 * Get registered steps
+	 * Get registered steps.
 	 *
-	 * @return array
+	 * @return array The registered steps.
 	 */
 	public function get_registered_steps(): array {
 		return $this->steps_map;
 	}
 
 	/**
-	 * Get trigger instance
+	 * Get trigger instance.
 	 *
 	 * @param string $trigger_slug The name of the trigger slug with which to retrieve the trigger class.
-	 * @return string
+	 * @return string The name of the trigger class.
 	 * @throws Automation_Exception Throws an exception if the step class does not exist.
```

Comment:
I'm not 100% sure since the WP Docs doesn't say anything but I feel like WP Core (the code) tends to add a new line above `@throws`

Reply:
I've seen it in different formats, before, together, with a blank line...